static/app/views/dashboards/detail.tsx

```diff
@@ -686,6 +689,17 @@ class DashboardDetail extends Component<Props, State> {
       }),
       {preventScrollReset: true}
     );
+
+    // Schedule stopping the timer for the end of the current task queue tick.
+    // This roughly corresponds to when React finishes the rendering and
+    // painting, and gives a decent idea of how long it took to open the Widget
+    // Builder
+    Promise.resolve().then(() => {
```

Comment:
There's a `scheduleMicroTask` function in https://github.com/getsentry/sentry/blob/58ab85a247cdf02e7f5f4d68a594abe338e69cf8/static/app/utils/useLocalStorageState.ts#L8C10-L8C27 which we probably should extract into a util and use instead.